Sources/CartonKit/Model/Template.swift

```diff
@@ -92,33 +92,33 @@ extension Template {
   ) throws {
     try fileSystem.writeFileContents(project.path.appending(component: "Package.swift")) {
       var content = """
-      // swift-tools-version:5.6
-      import PackageDescription
-      let package = Package(
-          name: "\(project.name)",\n
-      """
+        // swift-tools-version:5.6
```

Comment:
```suggestion
        // swift-tools-version:5.8
```
We could update the tools version for new projects.

Reply:
I think it should be changed after we updated the default toolchain version